### PR TITLE
First version

### PR DESCRIPTION
**Pull request by jsalbert**
First Version of keras-eval: Abstraction to perform keras model evaluations. 

- `evaluate`: Predict an image set and show metrics
- `predict`: Compute probabilities for a folder of images
- `predict_image`: Compute probabilities for a single image
- `get_metrics`: Get accuracy, sensitivity and precision values
- `plot_confusion_matrix`: Plot evaluation confusion matrix

### CODE REVIEW

**Comment by stephensolis** [comment]
Can you add a `.gitignore`? Some .DS_Store files were checked in

File: requirements.txt [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+pytest-cov
+flake8
+autoflake
+pep8
+autopep8
+
+stored
+backports.tempfile
+tensorflow
+scipy
```
Comment (stephensolis) [inline requirements.txt]: Just a note, the packages listed here should be required in tests only -- if you have packages required to run the code, they should be in the `setup.py` instead

**Comment by snormore** [comment]
We haven't done this very well in `keras-trainer` yet, but could you add a little blurb to the `README` about using `keras-eval`?

File: requirements.txt (hunk 1 above)
Comment (jsalbert) [reply requirements.txt]: So just test-related packages? 
Then should we remove `tensorflow` from the keras-trainer `requirements.txt`
